accessibility/core/build.gradle

```diff
@@ -7,7 +7,7 @@ android {
         targetSdkVersion 22
     }
 
-    compileSdkVersion 22
+    compileSdkVersion 23
     buildToolsVersion "22.0.1"
```

Comment:
targetVersion doesn't necessarily have to go with compileVersion.

what's buildToolsVersion supposed to be :tongue: I just went with lint :dancer: